qovery/adv_settings_model.go

```diff
@@ -0,0 +1,246 @@
+	for i := 0; i < v.NumField(); i++ {
+		tag := typeOfS.Field(i).Tag.Get("json")
+		key := tag[:strings.Index(tag, ",")]
+		_type := tfTypefromGoTtype(typeOfS.Field(i).Type)
+
+		advSettingsMap[key] = AdvSettingAttr{
+			Description: settingsDescription(key),
+			Type:        _type,
+		}
+	}
+
+	return advSettingsMap
+}
+
+func tfTypefromGoTtype(goType reflect.Type) attr.Type {
+	switch goType.String() {
+	case "*string", "string":
+		return tfTypes.StringType
+	case "*int", "int", "*int64", "int64", "*int32", "int32":
+		return tfTypes.Int64Type
+	case "*bool", "bool":
+		return tfTypes.BoolType
+	case "*[]string", "[]string":
+		return tfTypes.SetType{ElemType: tfTypes.StringType}
+	case "*map[string]string", "map[string]string":
+		return tfTypes.MapType{ElemType: tfTypes.StringType}
+	}
+
+	return tfTypes.ObjectType{AttrTypes: nil}
+}
+
+func clusterSettingsDescription(setting string) string {
+	switch setting {
+	case "aws.cloudwatch.eks_logs_retention_days":
+		return "Maximum retention days in Cloudwatch for EKS logs"
+	case "aws.eks.ec2.metadata_imds":
+		return "Specify the IMDS version you want to use. Possible values are Required (IMDS v2 only) and Optional (IMDS v1 and V2)"
+	case "aws.iam.admin_group":
+		return "Allows you to specify the IAM group name associated to the Qovery user"
+	case "aws.vpc.enable_s3_flow_logs":
+		return "Enable flow logs on the cluster VPC and store them in an s3 bucket"
+	case "aws.vpc.flow_logs_retention_days":
+		return "Set the number of retention days for flow logs"
+	case "cloud_provider.container_registry.tags":
+		return "Add additional tags on the cluster dedicated registry"
+	case "database.mongodb.allowed_cidrs", "database.mysql.allowed_cidrs", "database.postgresql.allowed_cidrs", "database.redis.allowed_cidrs":
+		return "List of allowed CIDRS"
+	case "database.mongodb.deny_public_access":
+		return "Deny public access to all MongoDB databases"
+	case "database.mysql.deny_public_access":
+		return "Deny public access to all MySQL databases"
+	case "database.postgresql.deny_public_access":
+		return "Deny public access to all PostgreSQL databases"
+	case "database.redis.deny_public_access":
+		return "Deny public access to all Redis databases"
+	case "load_balancer.size":
+		return "Allows you to specify the load balancer size in front of your cluster"
+	case "loki.log_retention_in_week":
+		return "Maximum Kubernetes pods (containers/application/jobs/cronjob) retention logs in weeks"
+	case "registry.image_retention_time":
+		return "Allows you to specify an amount in seconds after which images in the default registry are deleted"
+	case "pleco.resources_ttl":
+		return "Deprecated"
+	}
+
+	return noDescription
+}
+
+func autoScalingSettingsDescription(setting string) string {
+	switch setting {
+	case "hpa.cpu.average_utilization_percent":
+		return "CPU usage autoscaling trigger value"
+	}
+
+	return "No description for this service auto-scaling advanced setting."
```

Comment:
To be aligned with other:
```suggestion
	return noDescription
```
Or put contextual "No description for... " for each methods